exercises/practice/rotational-cipher/.meta/example.awk

```diff
@@ -0,0 +1,19 @@
+@load "ordchr"
+
+BEGIN {
+    FS = ""
+    OFS = ""
+    A = ord("A")
+    a = ord("a")
+}
+
+{
+    out = ""
+    for (i = 1; i <= NF; i++) {
+        if ("A" <= $i && $i <= "Z")
+            $i = chr((ord($i) - A + distance) % 26 + A)
```

Comment:
What's `distance`?

Reply:
The amount to rotate by. What would be a better name? Or should I have it in the first record?

Reply:
oh, it's the -v variable. That's the somewhat irritating thing about the standalone .awk files. Maybe we could come up with a comment convention so at least the word "distance" (or whatever for other exercises) shows up in search.